.openhands/microagents/test-style.md

```
@@ -0,0 +1,124 @@
+## 従来のテストコードからの移行方針
+
+### if文によるエラー検証の代替
+- 古い: `if err != nil { t.Fatalf("error: %v", err) }`
+- 新しい: `require.NoError(t, err)`
+
+### 値の比較の代替
+- 古い: `if actual != expected { t.Errorf("expected %v, got %v", expected, actual) }`
+- 新しい: `assert.Equal(t, expected, actual)`
+
+### ファイル存在確認の代替
+- 古い: `if _, err := os.Stat(path); os.IsNotExist(err) { t.Error("file not found") }`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)